web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -285,22 +285,36 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
         return (regexRes && regexRes[1].length > 0) ? regexRes[1].toTitleCase() : false;
     },
 
-    confirmDownload: function(collection, generateUrlCallbackScope, generateUrlCallback, params, textKey) {
+    confirmDownload: function(collection, generateUrlCallbackScope, generateUrlCallback, params) {
 
         var self = this;
 
         params = params || {};
         params.collection = collection;
 
         params.onAccept = function(callbackParams) {
+
             self.downloader.download(collection, generateUrlCallbackScope, generateUrlCallback, callbackParams);
+            var trackingAction = OpenLayers.i18n('downloadTrackingActionPrefix') + callbackParams.downloadLabel;
+
             trackDownloadUsage(
-                OpenLayers.i18n('downloadTrackingActionPrefix') + OpenLayers.i18n(textKey),
+                trackingAction,
                 collection.getTitle(),
                 self._getCollectionFiltersAsText(collection)
             );
-        };
 
+            if (callbackParams.emailAddress != undefined) {
+
+                var emailRequiredForDownload = callbackParams.collectEmailAddress != undefined ?
+                    "emailWasRequested" : "emailWasStoredInSession";
+
+                trackUserUsage(
+                    trackingAction,
```

Comment:
Scratch that, do we want to fire through to GA the email address or the collection? @pblain?